Projects/Core/Sources/Models/MakHoly/Reaction/LikeDetail.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LikeDetail.swift
+//  Core
+//
+//  Created by Eric Lee on 11/6/23.
+//  Copyright © 2023 com.tenten. All rights reserved.
+//
+
+import Foundation
+
+public struct LikeDetail {
+	
+	private let totalCount: Int
+	private let likeCount: Int
+	private let dislikeCount: Int
+	
+	public init(
+		totalCount: Int,
+		likeCount: Int,
+		dislikeCount: Int) {
+		self.totalCount = totalCount
+		self.likeCount = likeCount
+		self.dislikeCount = dislikeCount
+	}
+	
+	public init() {
+		self.totalCount = 0
+		self.likeCount = 0
+		self.dislikeCount = 0
+	}
```

Comment:
```suggestion
	public init(
		totalCount: Int = 0,
		likeCount: Int = 0,
		dislikeCount: Int = 0) {
		self.totalCount = totalCount
		self.likeCount = likeCount
		self.dislikeCount = dislikeCount
	}

```
p5
이런 방식은 처음 보긴 하는데, 이거랑 다른건가요?? 🧐

Reply:
이렇게 하면 초기화할때 두개중에 선택할 수 있어요
기본값으로 넣으면 초기화할 때 각 값들이 안뜨는게 기본이고 추가하고 싶으면 하나씩 직접 추가해야해요
그래서 Optional 사용하기 싫어서 초기값 때문에 필요할때는 두가지 이니셜라이저 분리해서 구현했어요

<img width="572" alt="스크린샷 2023-11-09 오후 2 37 17" src="https://github.com/DeveloperAcademy-POSTECH/MacC-Team-1010/assets/77574120/bcf38601-fc9a-4a45-b484-b06d4a40cd3a">

Reply:
오호.. 뭔가 convenience init으로도 처리가 가능할 거 같은데.. 한번 시도해보셔도 좋을 거 같아요! 이것도 좋습니다! 😄

Reply:
오 convenience init 메모 📝